eachdist.ini

```diff
@@ -7,6 +7,7 @@ ignore=
 sortfirst=
     opentelemetry-api
     opentelemetry-sdk
+    opentelemetry-instrumentation
```

Comment:
Shouldn't this go above `opentelemetry-sdk` since it depends on instrumentation?